Kernel/Arch/init.cpp

```diff
@@ -47,6 +47,7 @@
 #include <Kernel/FileSystem/VirtualFileSystem.h>
 #include <Kernel/Firmware/ACPI/Initialize.h>
 #include <Kernel/Firmware/ACPI/Parser.h>
+#include <Kernel/Firmware/DeviceTree/DeviceTree.h>
```

Comment:
(re commit description: fine as-is, but in general it's nice to make code move commits contain nothing but a mechanical code move. tweaks and additions can be in later commits.)